src/einsteinpy/utils/kerr_utils.py

```diff
@@ -331,3 +334,87 @@ def nonzero_christoffels():
         if chl[i][k][l]:
             vcl.append((i, k, l))
     return vcl
+
+
+def spin_factor(J, M, c):
+    """
+    Calculate spin factor(a) of kerr body
+
+    Parameters
+    ----------
+    J : float
+        Angular momentum in SI units(kg m2 s-2)
+    M : float
+        Mass of body in SI units(kg)
+    c : float
+        Speed of light
+
+    Returns
+    -------
+    float
+        Spin factor (J/(Mc))
+    
+    """
+    return J / (M * c)
+
+
+def event_horizon(Rs, a, theta=constant.pi_by_2, coord="BL"):
+    """
+    Calculate the radius of event horizon of Kerr black hole
+
+    Parameters
+    ----------
+    Rs : float
```

Comment:
Better variable name for schwarzschild radiius?

Reply:
R<sub>s</sub> is used everywhere in literature. Plus, it's been used in all other functions of `kerr_utils.py`